automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/PublishSpec.scala

```diff
@@ -116,5 +142,51 @@ class PublishSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Wo
         }
       }
     }
+
+  }
+
+  "As a user with no TCGA permissions" - {
+    "a published workspace" - {
+      "with TCGA control access" - {
+        "should see a information message about getting TCGA access" in withWebDriver { implicit driver =>
+          //log in as a Curator and create/publish TCGA workspace
+          val curatorUser = UserPool.chooseCurator
+          implicit val curatorAuthToken: AuthToken = curatorUser.makeAuthToken()
+
+          api.NIH.addUserToNIH(Config.Users.tcgaJsonWebToken)
```

Comment:
after talking with @breilly2 we think that this would be a good opportunity to harden NIH related tests and make them a lot more immune to side effects.

because some of the NIH group data lives outside of our direct control and is shared across all FiaBs in an environment, there could be some unreliability in these types of tests.

our proposal: 

there is a special authdomain pool of users. this test should select one of those users to do a fresh NIH link and then do the tests. specifically:

1) at the beginning of the test, forcibly remove every trace that the user has of being NIH-linked. we have admin endpoints to remove them from the TCGA-dbGaP-Authorized workbench group and also to remove their NIH information from thurloe
2) do the link
3) do the test
4) do step 1 again

could implement this as a `withFreshNihLink(jwt: String)` function or however you see fit. it would be great to start this pattern now so when we build up the NIH related tests, there's an expectation that they're reliable

Reply:
Yes, I think it's definitely worth making some improvements here. We could put it off for now, but I'm sure we'd never get back to it so now is really the time.